metis-normalization/src/test/java/eu/europeana/normalization/dates/extraction/extractors/MonthNameDateExtractorTest.java

```diff
@@ -0,0 +1,173 @@
+  }
+
+  void assertExtract(String input, String expected) {
+    final DateNormalizationResult dateNormalizationResult = PATTERN_MONTH_NAME_DATE_EXTRACTOR.extractDateProperty(input);
+    assertDateNormalizationResult(dateNormalizationResult, expected, DateNormalizationExtractorMatchId.MONTH_NAME);
+  }
+
+  private static Stream<Arguments> extractDayMonthYear() {
+    return Stream.of(
+        of("01 November 1989", "1989-11-01"),
+        of("32 November 1989", null),
+        of("01.November.1989", "1989-11-01"),
+        of("01,November,1989", "1989-11-01"),
+        //Combination of separators
+        of("01 November.1989", "1989-11-01"),
+        of("01 November,1989", "1989-11-01"),
+        of("01.November 1989", "1989-11-01"),
+        of("01.November,1989", "1989-11-01"),
+        of("01,November 1989", "1989-11-01"),
+        of("01,November.1989", "1989-11-01"),
+
+        //Some other languages or name formats
+        of("01 nov. 1989", "1989-11-01"),
+        of("01 ное 1989", "1989-11-01"),
+        of("01 Νοεμβρίου 1989", "1989-11-01"),
+        of("01 January 1989", "1989-01-01"),
+        of("01 Νοεμβρίου 1989", "1989-11-01"),
+        of("01 νοεμβρίου 1989", "1989-11-01"),
+        of("01 ΝΟΕΜΒΡΊΟΥ 1989", "1989-11-01"),
+        //Italian
+        of("01 Novembre 1989", "1989-11-01"),
+
+        //Incorrect month
+        of("99 November 9989", null),
+        of("99 November 9989", null),
+
+        //Too few digits on year
+        of("1 January 989", null),
+        of("1.January.989", null),
+        of("1,January,989", null),
+        //Too many digits on year
+        of("01 January 12345", null),
+        //Too many digits on day
+        of("123 January 1234", null),
+
+        //Other invalids
+        //Double spaces should not match
+        of("1989  November  01", null),
+        //Double dots should not match
+        of("1989..November..01", null),
+        //Double commas should not match
+        of("1989,,November,,01", null)
+    );
+  }
+
+  private static Stream<Arguments> extractMonthDayYear() {
+
+    return Stream.of(
+        of("November 01 1989", "1989-11-01"),
+        of("November 32 1989", null),
+        of("November.01.1989", "1989-11-01"),
+        of("November,01,1989", "1989-11-01"),
+        //Combination of separators
+        of("November 01.1989", "1989-11-01"),
+        of("November 01,1989", "1989-11-01"),
+        of("November.01 1989", "1989-11-01"),
+        of("November.01,1989", "1989-11-01"),
+        of("November,01 1989", "1989-11-01"),
+        of("November,01.1989", "1989-11-01"),
+
+        //Some other languages or name formats
+        of("nov. 01 1989", "1989-11-01"),
+        of("ное 01 1989", "1989-11-01"),
+        of("January 01 1989", "1989-01-01"),
+        of("Νοεμβρίου 01 1989", "1989-11-01"),
+        of("νοεμβρίου 01 1989", "1989-11-01"),
+        of("ΝΟΕΜΒΡΊΟΥ 01 1989", "1989-11-01"),
+        //Italian
+        of("Novembre 01 1989", "1989-11-01"),
+
```

Comment:
only this case still fails
```
of("listopad 01 1989", "1989-11-01"), //Polish
```
results:
```
Expected :1989-11-01
Actual   :1989-10-01
```

Reply:
You are right! I assumed it worked, but now that I checked further there seems to be an ambiguous case.. In Croatian is October but in Polish is November.
I will create a ticket to discuss this with Nuno on a further update.